src/components/utils/mui-select-input.js

```diff
@@ -0,0 +1,35 @@
+import { useController } from 'react-hook-form';
+import { InputLabel, MenuItem, Select } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import FormControl from '@mui/material/FormControl';
+
+const MuiSelectInput = ({ name, options, label, ...props }) => {
+    const {
+        field: { value, onChange },
+    } = useController({
+        name,
+    });
+
+    return (
+        <FormControl fullWidth>
```

Comment:
I don't understand what is done here with RHF ? Why don't you just use SelectInput from GridStudty ?